src/toast/hoc/toast.jsx

```diff
@@ -21,9 +22,19 @@ const addDelayBeforeRemove = defaultDelay => (WrappedComponent) => {
     }
 
     render() {
-      return <WrappedComponent {...this.props} />
+      return <WrappedComponent {...this.props} print={!!this.props.title} />
```

Comment:
add a comment saying that if there is not title this is because the toast is empty since title is mandatory